changelog.d/9649.bugfix

```diff
@@ -0,0 +1 @@
+Fixed Code Quality Issues and improved code quality.
```

Comment:
This is a `misc` change, there's no user-visible bug that is fixed by these changes.

Reply:
@clokep updated the suggested changes.